e2e/tests/components/menu/menu-item-focus/menu-item-focus.po.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { $, $$, browser } from 'protractor';
+
+export class MenuItemFocusPage {
+
+    menuTrigger = $('#open-menu');
+    topFocus = $('#top-focus');
+    cdkOverlayContainer = $('.cdk-overlay-container');
+    menu = $('.ux-menu');
+    disableCheckbox = $('#disable-checkbox');
+
+    // Component Inputs
+    toggleSwitch = $('.ux-menu ux-toggleswitch');
+    checkboxes = $$('.cdk-overlay-container ux-checkbox');
+    radioButtons = $$('ux-radio-button');
```

Comment:
FYI you can add classes or IDs to components in your test page to make finding them a lot easier.